chains/smartbch/logo.svg

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="230" height="230">
```

Comment:
Not sure you need these width and height

Can minimify probably the logo with https://jakearchibald.github.io/svgomg/